recipes/bornagain/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "BornAgain" %}
+{% set version = "1.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://apps.jcns.fz-juelich.de/src/BornAgain/BornAgain-{{ version }}.tar.gz
+  sha256: 20e390b683b3036aabf48d266e5e74da46ce9d781e72379dccb3d227ed300d40
+
+build:
+  number: 0
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - cmake
+    - ninja
+
+  host:
+    - boost
+    - fftw
+    - gsl
+    - qt 5.9.*
+    - python >=3
```

Please remove the ``>=3`` as you are already skipping python 2 this is not necessary
```suggestion
    - python
```